spec/features/hiring_staff_can_sign_out_spec.rb

```diff
@@ -8,6 +8,7 @@
     visit root_path
 
     click_on(I18n.t('nav.sign_out'))
+    visit auth_dfe_signout_path
```

Comment:
In both specs, the `click_on` should cause the `visit`, right? We don't want the user to need to type in the `auth_dfe_signout_path` themselves.

Reply:
I've now made this change - see next commit.